content/docs/sidebar.json

```diff
@@ -562,7 +562,7 @@
             "slug": "prepare-repositories"
           },
           {
-            "label": "Views",
+            "label": "Views and Experiments",
```

Comment:
Q: So why do we want to bundle Views and Experiments topics in a single section? Maybe create an **Experiments** section instead to contain Explore ML Experiments, Visualize and Compare, and Run Experiments?

Reply:
I added it within the same menu coz everything you do in terms of experiments is within the view. Exploring experiments is the same as exploring a view. So, couldn't draw a clear boundary between views and experiments. This is different from `Teams`, `Accounts` etc which are clearly different. But let me know if it's better to separate them and I can move the pages as you have suggested.

Reply:
Up to you on this. I'm not familiar enough with this region of the docs still, unfort.